doc/060_forget.rst

```diff
@@ -282,3 +317,52 @@ last-day-of-the-months (11 or 12 depends if the 5 weeklies cross a month).
 And finally 75 last-day-of-the-year snapshots. All other snapshots are
 removed.
 
+Customize pruning
+*****************
+
+To understand the custom options, we first explain how the pruning process works:
+
+- First all snapshots and directories within snapshots are scanned to get a list
+  of which data is used. This can be done pretty fast.
+- Then for all pack files ``prune`` finds out if the file is fully used, partly
```

Comment:
Simple and clear language should always take precedence in user manuals, e.g. get, not retrieve/use, not utilize/change, not modify.

In this case I'd stick with `finds out if` since it's more understandable than your suggested improvement. My friends from other lines of work don't know terms like iterate, increment that's why you should always distinguish between your audiences and choose your wording accordingly. 

PS: I'm member of the Write The Docs Berlin user group :-)

Reply:
I didn't change the "finds out"